content/support/enablement/k8-migration.md

```diff
@@ -0,0 +1,101 @@
+# How to Migrate Sourcegraph from one cluster to another.
+
+The instructions in this document will guide you on the steps you need to take to migrate your Sourcegraph deployment from one Cluster to another. This is a process that involves the backing up of the Sourcegraph database by generating dumps and restoring the dumps in the newly deployed Sourcegraph instance on another cluster.
+
+> WARNING: **Only core data will be backed up**.
+>
+> These instructions will only back up core data including user accounts, configuration, repository-metadata, etc. Other data will be regenerated automatically:
+>
+> - Repositories will be re-cloned
+> - Search indexes will be rebuilt from scratch
+>
+> The above may take a while if you have a lot of repositories. In the meantime, searches may be slow or return incomplete results. This process rarely takes longer than 6 hours and is usually **much** faster.
+
+> NOTE: In some places you will see `$NAMESPACE` used. Add `-n $NAMESPACE` to commands if you are not using the default namespace
+> More kubectl configuration options can be found here: [kubectl Cheat Sheet](https://kubernetes.io/docs/reference/kubectl/cheatsheet/)
+
+## Overview
+
+1. Backup Sourcegraph databases in the old cluster by generating dumps
+2. Restore the Dumps in the freshly deployed instance on another cluster.
+
+### Steps to backup Sourcegraph Databases
+
+In this section of the tutorial, you would learn how to back up the primary `sourcegraph` database and the `codeintel` database.
+
+- Verify that the deployment is running.
+
+  `kubectl get pods -A`
+
+- Stop all connections to the database by removing the frontend deployment.
+
+  `kubectl scale --replicas=0 deployment/sourcegraph-frontend`
+
+- Generate the database dumps.
+
+  - Dumps for the primary **sourcegraph** DB
+    `kubectl exec -it $pgsql_POD_NAME -- bash -c 'pg_dump -C --username sg sg' > sourcegraph_db.out`
+
+  - Dumps for the **codeintel** DB
+    `kubectl exec -it $codeintel-db_POD_NAME -- bash -c 'pg_dump -C --username sg sg' > codeintel_db.out`
+
+- Ensure you copy/move the `sourcegraph_db.out` and `codeintel_db.out` files to the root of the `deploy-sourcegraph` directory of the new Sourcegraph deployment.
+
+### Steps to restore Sourcegraph Databases into a new environment
+
+In this section of the tutorial, you would learn how to restore the primary `sourcegraph` database and the `codeintel` database from the generated dumps in the previous section.
+
+- Before restoring on the new deployment, It is very important you stop all connections to the Database from the frontend.
+
+  -     Scale down all the pods in the deployment
+    `kubectl scale deployment ---all --reliplicas=0 `
```

Comment:
Hello! I noticed a few places with `reliplicas`. I'm guessing you meant `replicas` ?